### PR TITLE
add basic skeleton for image model

### PR DESCRIPTION
**Pull request by jamesgorrie**
I had a bit of trouble getting images from contributors into their respective components.

It felt like we have a few established patterns, so just going to try and whittle down what we have into those.

Feedback welcome.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@davidpmccormick as much as I would like to take the approval, I've decided to implement it to test it.

Basically we've got:
`Image`
`CaptionedImage`
`UiImage`
`UiCaptionedImage`

It seems the pattern is emerging that we need stuff for parsing (the non UI models) and then UiModels for things at render time.

I quite like it.

**Comment by jamesgorrie** [comment]
Ping @gestchild as it's out for approval again.

File: common/services/prismic/parsers.js [hunk 1]
```diff
@@ -7,9 +7,22 @@ import type { Tasl } from '../../model/tasl';
 import type { LicenseType } from '../../model/license';
 import type { Place } from '../../model/place';
 import type { BackgroundTexture, PrismicBackgroundTexture } from '../../model/background-texture';
+import type { CaptionedImageProps } from '../../views/components/Images/Images';
 import { licenseTypeArray } from '../../model/license';
 import { parseInfoPage } from './info-pages';
 
+const placeHolderImage = {
+  contentUrl: 'https://via.placeholder.com/160x900?text=Placeholder',
+  width: 160,
```
Comment (davidpmccormick) [inline common/services/prismic/parsers.js]: should this be `1600`?

File: common/services/prismic/parsers.js [hunk 2]
```diff
@@ -7,9 +7,22 @@ import type { Tasl } from '../../model/tasl';
...
+  height: 900,
+  alt: 'Placeholder image',
+  tasl: {
+    contentUrl: 'https://via.placeholder.com/160x900?text=Placeholder',
```
Comment (davidpmccormick) [inline common/services/prismic/parsers.js]: and `1600` here?

**Pull request merged**